src/test/java/org/testeditor/fixture/web/WebDriverFixtureTest.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2018 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.web;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.openqa.selenium.WebElement;
+import org.testeditor.fixture.core.FixtureException;
+
+
+public class WebDriverFixtureTest {
```

Comment:
I guess you could add test cases for when the elements are not found, right?

Reply:
done ;O) thanx